### ISSUE TITLE
Provide additional data and setup instructions for local users

### ISSUE THREAD

**Issue by samnooij** [carpentries-lab/metagenomics-analysis#70]
Thank you for sharing this awesome lesson material! I'm preparing to teach this workshop as an instructor and I'm running into a little issue. I'm going to teach the final part with RStudio. Therefore, I need the Kraken BIOM file as input. Also, I do not have the AWS virtual machine available right now, so I'm working on my PC. This is where I needed to get creative.

First, I could not find the Kraken output file, so I decided to run that myself. Then, I found that there were no installation instructions for Kraken (and kraken-biom!). I ended up installing all software using the conda YAML file copied at the bottom. (I used the Kraken database linked in the lesson material: `ftp://ftp.ccb.jhu.edu/pub/data/kraken2_dbs/old/minikraken2_v2_8GB_201904.tgz`.)
Then I tried as input both the subsampled files from `sub.tar.gz` on https://figshare.com/articles/dataset/Data_Carpentry_Genomics_beta_2_0/7726454 and the untrimmed reads from `backup.tar.gz`. Neither of these allows me to produce results in R identical to the lesson material. (But it does work and I can run the R commands!) Would it be possible to share the BIOM file and/or the actual input data used to create the output as shown in the lessons? And maybe add a little more instructions for the 'do it yourself users'? Thanks in advance!

```yaml
name: metagenomics-workshop

channels:
 - bioconda
 - conda-forge
 - defaults

dependencies:
 - fastqc=0.11.7
 - trimmomatic=0.38
 - bwa=0.7.17
 - samtools=1.9
 - bcftools=1.9
 - kraken2=2.1.2
 - kraken-biom=1.2.0
```

**Comment by nselem** [comment]
Thank you for using the material. We do not use Kraken in the lesson since because of RAM limitations some laptops won't be able to run it. For that reason, we provide the report files from Kraken, and you only will need kraken-biom. Anyway, here is the biom file, which also should be available for future versions of the data in zenodo 
https://github.com/carpentries-incubator/metagenomics/blob/gh-pages/files/cuatroc.biom

Some notes: 
1. The kraken output files are on the zenodo data as indicated in the data section
https://carpentries-incubator.github.io/metagenomics-workshop/data/index.html

2. The software can be installed using the yml [specifications file](https://github.com/carpentries-incubator/metagenomics/blob/gh-pages/files/spec-file-Ubuntu22.txt) indicated in the setup section

**Comment by samnooij** [comment]
Thank you for sharing the BIOM file! I just noticed that I had been looking at the setup page for the genomics workshop (https://datacarpentry.org/genomics-workshop/setup.html) instead of the one for the metagenomics workshop (https://carpentries-incubator.github.io/metagenomics-workshop/setup.html). My bad! I see the metagenomics setup page indeed includes all the software packages and a conda YAML file. That should settle the issues I ran into. Sorry if I caused some confusion.
